docs/installation.md

```diff
@@ -85,7 +85,7 @@ $ kubectl get pods --namespace=nginx-ingress
 
 ## 4. Get Access to the Ingress Controller
 
-**If you created a daemonset**, ports 80 and 443 of the Ingress controller container are mapped to the same ports of the node where the container is running. To access the Ingress controller, use those ports and an IP address of any node of the cluster where the Ingress controller is running.
+**If you created a daemonset**, ports 80 and 443 of the Ingress controller container are mapped to the same ports of the node where the container is running. To access the Ingress controller, use those ports and an IP address of any node of the cluster where the Ingress controller is running. Opening HTTP ports node on the GKE firewall may require changing the node template, see https://github.com/nginxinc/kubernetes-ingress/issues/502
```

Comment:
Using port mapping on GKE is uncommon - typically people would use a service with type LoadBalancer. That’s why GKE specific advice here doesn’t make sense to me.

We can mention something like “Make sure that the firewall allows traffic on those ports”.

What do you think?